src/utils/short-url-generator.ts

```diff
@@ -0,0 +1,12 @@
+export function generateShortUrl(): string {
+  const characters = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789';
+  let shortUrl = '';
+  // Generate a random 6-character string with upper and lower case letters and numbers.
+  for (let i = 0; i < 6; i += 1) {
+    shortUrl += characters[Math.floor(Math.random() * characters.length)];
+  }
+  return shortUrl;
+}
```

Comment:
Shorter and more modern.

```suggestion
export function generateShortUrl(): string {
  const characters = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789';
  return Array.from({ length: 6 }, () => characters[Math.floor(Math.random() * characters.length)]).join('');
}
```

Reply:
Noted sir, but i noticed the last time I tried using Array.____, I was having lint error of my module not update to date.